es/news/_posts/2023-05-12-ruby-3-3-0-preview1-released.md

```diff
@@ -0,0 +1,173 @@
+---
+layout: news_post
+title: "Publicado Ruby 3.3.0-preview1"
+author: "naruse"
+translator: vtamara
+date: 2023-05-12 00:00:00 +0000
+lang: es
+---
+
+{% assign release = site.data.releases | where: "version", "3.3.0-preview1" | first %}
+
+Nos complace anunciar la publicación de Ruby {{ release.version }}. Ruby 3.3
+añade un nuevo compilador JIT en ruby puro llamado RJIT, usa lrama
+como generador del analizador sintáctico así como muchas mejoras
+de desempeño especialmente de YJIT.
+
+## RJIT
+
+* Se introdujo RJIT un compilador JIT en Ruby puro que remplaza a MJIT.
+  * RJIT soporta sólo la arquitectura x86\_64 en plataformas Unix.
+  * A diferencia de MJIT, no require de un compilador C en tiempo de ejecución.
+* RJIT existe sólo con propósitos experimentales.
+  * Debe seguir usando YJIT en producción.
+* Si le interesa desarrollar un compilador JIT para Ruby, por favor
+  revise la [presentación de k0kubun en el día 3 de RubyKaigi](https://rubykaigi.org/2023/presentations/k0kubun.html#day3).
+
+## Usa Lrama en lugar de Bison
+
+* Remplaza Bison con [el generador de analizadores sintácticos LALR Lrama](https://github.com/yui-knk/lrama) [Feature #19637]
+  * Si tiene interés por favor vea [la visión futura de los analizadores sintácticos de Ruby](https://rubykaigi.org/2023/presentations/spikeolaf.html)
+
+## YJIT
+
+* Mejoras en desempeño significativas respecto a 3.2
+  * Mejorado el soporte para argumentos splat y para el resto de argumentos.
```

Comment:
I would like to meet with the @ruby/www-ruby-lang-org-i18n-es team members and discuss how we can handle this kind of scenario (Ruby splat operator) where we don't have a literal translation to Spanish.

Reply:
What about proposing in the Glossary: https://github.com/ruby/www.ruby-lang.org/wiki/English-Spanish-Glossary